pkg/cmd/server/origin/master_config.go

```diff
@@ -141,10 +151,25 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 	admissionControlPluginNames := []string{"OriginNamespaceLifecycle"}
 	admissionController := admission.NewFromPlugins(privilegedLoopbackKubeClient, admissionControlPluginNames, "")
 
+	var tokenGetter serviceaccount.ServiceAccountTokenGetter
+	if options.KubernetesMasterConfig == nil {
```

Comment:
I wasn't sure if the loopback client would ever be authenticated using a service account token, so was trying to prevent ever calling our own auth layer to validate a service account token with external kubernetes